h2o-core/src/main/java/water/api/ModelsHandler.java

```diff
@@ -161,6 +162,23 @@ public StreamingSchema fetchMojo(int version, ModelsV3 s) {
     return new StreamingSchema(model.getMojoStream(), filename);
   }
 
+  @SuppressWarnings("unused") // called from the RequestServer through reflection
+  public JobV3 makePDP(int version, PDPV3 s) {
+    PDP pdp;
+    if (s.destination_key != null)
+      pdp = new PDP(s.destination_key.key());
+    else
+      pdp = new PDP(Key.<PDP>make());
+    s.fillImpl(pdp); //fill frame_id/model_id/nbins/etc.
+    return new JobV3(pdp.execImpl());
+  }
+
+  @SuppressWarnings("unused") // called from the RequestServer through reflection
+  public PDPV3 fetchPDP(int version, KeyV3.PDPKeyV3 s) {
```

Comment:
Did you test this endpoint? Can the server convert `{name}` in the endpoint's URL into a proper `KeyV3.PDPKeyV3` object?